src/components/TextInput/TextInput.vue

```diff
@@ -78,6 +82,23 @@
         @input="onInput"
         @focus="onFocus"
       >
+      <div
+        v-if="showClearButton"
+        :class="['pr-2 pt-3 pb-3 text-gray-500', {'!pr-1 !py-2': small}]"
+      >
+        <lob-button
```

Comment:
nit that I found while trying to help with the autofill issue: we're not importing/declaring this component, so a warning is thrown in the console. looks like we're using a normal `button` later in the component - wondering if there's any thoughts on using a lobbutton vs. a normal button here? it's not like we need the styling, really... ☺️

Reply:
yeah I don't think we _have_ to make _all_ buttons `LobButton`s.  If no styling or functionality is needed that the `LobButton` provides I think a plain ole `<button>` is fine.

Reply:
Changed it to button